src/test/java/uk/sky/cqlmigrate/CqlMigratorConsistencyLevelIntegrationTest.java

```diff
@@ -0,0 +1,148 @@
+import static org.scassandra.http.client.PrimingRequest.then;
+import static org.scassandra.http.client.types.ColumnMetadata.column;
+import static org.scassandra.matchers.Matchers.containsQuery;
+
+public class CqlMigratorConsistencyLevelIntegrationTest {
+
+    @Rule
+    public final ScassandraServerRule scassandra = new ScassandraServerRule(FREE_PORT, 1235);
+
+    private static final String[] CASSANDRA_HOSTS = {"localhost"};
+    public static final int FREE_PORT = PortScavenger.getFreePort();
+    private static final String CLIENT_ID = UUID.randomUUID().toString();
+    private static final String TEST_KEYSPACE = "cqlmigrate_test";
+
+    public static final ConsistencyLevel EXPECTED_READ_CONSISTENCY_LEVEL = ConsistencyLevel.LOCAL_ONE;
+    public static final ConsistencyLevel EXPECTED_WRITE_CONSISTENCY_LEVEL = ConsistencyLevel.ALL;
+
+    private final CassandraLockConfig lockConfig = CassandraLockConfig.builder()
+            .withClientId(CLIENT_ID)
+            .build();
+
+    private final CqlMigratorImpl migrator = new CqlMigratorImpl(
+            CqlMigratorConfig.builder()
+                    .withCassandraLockConfig(lockConfig)
+                    .withReadConsistencyLevel(EXPECTED_READ_CONSISTENCY_LEVEL)
+                    .withWriteConsistencyLevel(EXPECTED_WRITE_CONSISTENCY_LEVEL)
+                    .build()
+    );
+
+    private PrimingClient primingClient = scassandra.primingClient();
+    private ActivityClient activityClient = scassandra.activityClient();
+    private static Session session;
+
+    @Before
+    public void initialiseLockTable() throws ConfigurationException, IOException, TTransportException, InterruptedException {
+        session = Cluster.builder()
+                .addContactPoints(CASSANDRA_HOSTS)
+                .withPort(FREE_PORT)
+                .build()
+                .connect();
+
+        primingClient.prime(preparedStatementBuilder()
+                .withQuery("INSERT INTO cqlmigrate.locks (name, client) VALUES (?, ?) IF NOT EXISTS")
+                .withThen(
+                        then()
+                                .withVariableTypes(PrimitiveType.TEXT, PrimitiveType.TEXT)
+                                .withColumnTypes(column("client", PrimitiveType.TEXT), column("[applied]", PrimitiveType.BOOLEAN))
+                                .withRows(ImmutableMap.of("client", CLIENT_ID, "[applied]", true)))
+        );
+
+        primingClient.prime(preparedStatementBuilder()
+                .withQuery("DELETE FROM cqlmigrate.locks WHERE name = ? IF client = ?")
+                .withThen(
+                        then()
+                                .withVariableTypes(PrimitiveType.TEXT, PrimitiveType.TEXT)
+                                .withColumnTypes(column("client", PrimitiveType.TEXT), column("[applied]", PrimitiveType.BOOLEAN))
+                                .withRows(ImmutableMap.of("client", CLIENT_ID, "[applied]", true)))
+        );
```

Comment:
I don't know enough about scassandra, but is it possible to make these stubs less specific? Otherwise we're tying the test very closely to the implementation details of the queries.